docs/sources/administration/data-source-management/team_lbac.md

```diff
@@ -0,0 +1,156 @@
+---
+aliases:
+  - ../datasources/datasource_permissions/team_lbac
+  - ../datasources/datasource_permissions/teamlbac
+  - ../enterprise/datasource_permissions/team_lbac
+  - ../enterprise/datasource_permissions/teamlbac
+  - ../permissions/datasource_permissions/team_lbac
+  - ../permissions/datasource_permissions/teamlbac
+description: Data source permissions for Teams for Grafana administrators
+labels:
+  products:
+    - enterprise
+    - cloud
+title: Data source permissions for Team LBAC
+weight: 100
+---
+
+# Team LBAC
+
```

Comment:
Suggest we add a public preview note like for here https://grafana.com/docs/grafana-cloud/connect-externally-hosted/multi-stack-data-sources/